skyvern/webeye/models.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+from skyvern.forge.sdk.schemas.persistent_browser_sessions import PersistentBrowserSession
+
+
+class BrowserSessionResponse(BaseModel):
+    session_id: str
+    organization_id: str
+    runnable_type: str | None = None
+    runnable_id: str | None = None
+    created_at: datetime
+    modified_at: datetime
+    deleted_at: datetime | None
+
+    @classmethod
+    def from_browser_session(cls, browser_session: PersistentBrowserSession) -> "BrowserSessionResponse":
```

Comment:
https://stackoverflow.com/questions/61544854/from-future-import-annotations

can do `from __future__ import annotations` and this becomes:
```suggestion
    def from_browser_session(cls, browser_session: PersistentBrowserSession) -> BrowserSessionResponse:
```

Reply:
Nice one 👍